.github/workflows/ipr.yml

```
@@ -0,0 +1,20 @@
+name: 'ecma-262'
+
+on: [pull_request_target, push]
+
+jobs:
+  build:
+    name: 'check IPR form'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```

```suggestion
      - uses: actions/checkout@v2
        with:
          fetch-depth: 0
```

The default only checks out the last commit on the branch.